ppafm/cli/relaxed_scan.py

```diff
@@ -1,37 +1,39 @@
 #!/usr/bin/python -u
 
+import itertools as it
 import os
 
 import numpy as np
 
-import ppafm as PPU
-import ppafm.core as PPC
-import ppafm.cpp_utils as cpp_utils
-import ppafm.GridUtils as GU
-import ppafm.HighLevel as PPH
-from ppafm import io
+from .. import common, core, io
+from ..GridUtils import interpolate_cartesian
+from ..HighLevel import perform_relaxation
 
 
-def rotVec( v, a ):
-    ca=np.cos(a);      sa=np.sin(a)
-    x=ca*v[0]-sa*v[1]; y=sa*v[0]+ca*v[1];
-    v[0]=x; v[1]=y;
+def rotate_vector(v, a):
+    ca = np.cos(a)
+    sa = np.sin(a)
+    x = ca * v[0] - sa * v[1]
+    y = sa * v[0] + ca * v[1]
+    v[0] = x
+    v[1] = y
     return v
 
-def rotFF( Fx,Fy, a ):
-    ca=np.cos(a);      sa=np.sin(a)
-    Fx_=ca*Fx-sa*Fy; Fy_=sa*Fx+ca*Fy;
-    return Fx_,Fy_
 
+def rotate_ff(fx, fy, a):
+    ca = np.cos(a)
+    sa = np.sin(a)
+    new_fx = ca * fx - sa * fy
+    new_fy = sa * fx + ca * fy
+    return new_fx, new_fy
 
-def main():
 
-    parser = PPU.CLIParser(
-        description='Perform a scan, relaxing the probe particle in a precalculated force field. '
-            'The generated force field is saved to Q{charge}K{klat}/OutFz.xsf.'
+def main():
+    parser = common.CLIParser(
+        description="Perform a scan, relaxing the probe particle in a precalculated force field. " "The generated force field is saved to Q{charge}K{klat}/OutFz.xsf."
```

Comment:
Can be combined into a single string.

Reply:
done in https://github.com/Probe-Particle/ppafm/pull/215/commits/b1616f1d8ea33d0c1c2074bd5a9f4b56ecd00bf5